src/Utility.js

```diff
@@ -92,6 +92,18 @@ const __class = lang.setObject('crm.Utility', lang.mixin({}, Utility, {
     }
     return id;
   },
+  trimText: function trimText(text = {}, wordCount = {}) {
+    const words = text.split(' ');
+    if (words.length > wordCount) {
+      const intermediate = words.slice(0, wordCount);
+      if (intermediate[wordCount - 1].slice(-1) === '.') {
```

Comment:
Babel polyfills es6 string methods, so instead of slice(-1) you can use `endsWith('.')`